dashboard/config/linux/upstream-apparmor-kasan.config

```diff
@@ -9816,11 +9879,13 @@ CONFIG_KCOV=y
 CONFIG_KCOV_ENABLE_COMPARISONS=y
 CONFIG_KCOV_INSTRUMENT_ALL=y
 CONFIG_KCOV_IRQ_AREA_SIZE=0x40000
+# CONFIG_KCOV_SELFTEST is not set
```

Comment:
Perhaps we want to enable this? This adds several hundreds ms on boot, but prevents interrupt issues. We don't have any other good way to detect them.